### PR TITLE
feat: display instructlab sessions, with mock instructlab manager

### LINKED ISSUE #1568: Implement InstructLab sessions screen
### Is your enhancement related to a problem? Please describe

If there are session, display then as a table.

### Describe the solution you'd like

See the **Model Details Knowledges & Skills (0)** screen on https://design.penpot.app/#/workspace/971a4169-b56c-80d0-8002-4ff987ca8805/6e12068a-09a0-801a-8004-24b355541870?page-id=45387374-aad3-80a3-8004-c8e259891437&layout=layers

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### LINKED ISSUE #1616: Provide a mock InstructLab manager and connect it to the InstructLab session screen

### PR DESCRIPTION
**Pull request by feloy**
### What does this PR do?

display instructlab sessions, with mock instructlab manager

### Screenshot / video of UI

https://github.com/user-attachments/assets/4e95c681-a8f8-4735-9473-d58d23848979

### What issues does this PR fix or reference?

Fixes #1568
Fixes #1616

### How to test this PR?

You need to set an hidden setting, in `~/.local/share/containers/podman-desktop/configuration/settings.json`

(you must completely stop Podman Desktop - not only close the window -, edit the config file, then restart Podman Desktop for the change to be taken into consideration) 

```
  "ai-lab.experimentalTuning": true,
```

### CODE REVIEW

File: packages/backend/src/studio.ts [hunk 1]
```diff
@@ -49,6 +49,7 @@ import { RecipeManager } from './managers/recipes/RecipeManager';
 import { GPUManager } from './managers/GPUManager';
```
Comment (axel7083) [inline packages/backend/src/studio.ts:49]: I think we should create a dedicated instrulab-api-impl.ts object, and also registering it in the same way we are doing for studio-api-impl. 

As we do not know yet the implication and complexity of instrutlab implementation, it is probably better to keep it appart.

Comment (feloy) [reply packages/backend/src/studio.ts:49]: You mean to also split the `SutdioAPI`, by having a separate `InstructlabAPI`?

Comment (axel7083) [reply packages/backend/src/studio.ts:49]: Yeah !

Comment (feloy) [reply packages/backend/src/studio.ts:49]: ok, I'll make the change

File: packages/backend/src/instructlab-api-impl.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
+/**********************************************************************
```
Comment (axel7083) [inline packages/backend/src/instructlab-api-impl.ts:1]: Thanks ! Great job

**Review by axel7083 — APPROVED** [review]
LGTM ! :rocket: nice job

**Review by jeffmaury — COMMENTED** [review]
LGTM. 

Could we get navigation and tooltip from the model column to model details ?

**Comment by feloy** [comment]
> LGTM.
> 
> Could we get navigation and tooltip from the model column to model details ?

That should be ok now

**Comment by feloy** [comment]
> Looks better to me. I expected also the same navigation on target model but could be a follow up PR

As the target model will exist only when the instructlab session will be finished, I prefered not to add it for now.

**Pull request merged**